infrastructure/code/bundle.go

```diff
@@ -200,14 +200,12 @@ func (b *Bundle) addCodeActions(ctx context.Context, issues []snyk.Issue) ([]sny
 				// We only allow the issues whose file extensions are supported by the
 				// backend.
 				supported, err := b.isAutofixSupportedForExtension(ctx, issues[i].AffectedFilePath)
-				if err != nil {
-					return []snyk.Issue{}, err
-				}
-
-				if supported {
-					issues[i].CodeActions = append(issues[i].CodeActions, *b.createDeferredAutofixCodeAction(ctx, issues[i]))
-				} else {
-					log.Debug().Str("method", "addCodeActions").Msg("Autofix is not supported for " + issues[i].AffectedFilePath + " file extension.")
+				if err == nil {
```

Comment:
Why did you invert that logic? Checking `err` against `nil` is unintuitive, as it is usually the other way round.

Reply:
I did invert, since we fail the whole analysis currently, if we fail to verify supported extensions. Refactored the logic a little to be more intuitive, please have a look.